scadalts-ui/src/router.js

```diff
@@ -41,87 +43,94 @@ const routing = new Router({
 			name: 'about',
 			component: About,
 			meta: {
-				requiresAuth: true
+				requiresAuth: true,
 			},
 		},
 		{
 			path: '/alarms',
 			name: 'alarms',
-			component: Alarms,
+			component: AlarmTabs,
 			meta: {
-				requiresAuth: true
+				requiresAuth: true,
 			},
+			children: [
+				{
+					path: 'plc',
+					component: Alarms,
+				},
+				{
+					path: 'plc-history',
+					component: HistoricalAlarmsComponent,
+				},
+				{
+					path: 'scada',
+					name: 'scada',
+					component: EventList,
+				},
+			],
```

Comment:
Don't worry about that children. If you press F5 to refresh one of this page you lose the "logged user state" but in further versions of Modern Interface that bug has been fixed so it works.

Reply:
ok